csvProcessor/src/main/java/org/incredible/csvProcessor/Main.java

```diff
@@ -169,18 +161,67 @@ private static void setCertModelsList(CSVParser csvParser) {
     /**
      * to generateQRCode for certificate
      **/
-    private static void generateQRCodeForCertificate(Assertion assertion) {
+    private static void generateQRCodeForCertificate(CertificateExtension certificateExtension) {
+
+        List<String> text = new ArrayList<>();
+        List<String> data = new ArrayList<>();
+        List<String> filename = new ArrayList<>();
+
+        text.add(certificateExtension.getRecipient().getName());
+
+        data.add(certificateExtension.getBadge().getName());
+        filename.add(certificateExtension.getId());
+        QRCodeGenerationModel qrCodeGenerationModel = new QRCodeGenerationModel();
+        qrCodeGenerationModel.setText(text);
+        qrCodeGenerationModel.setFileName(filename);
+        qrCodeGenerationModel.setFileFormat("png");
+        qrCodeGenerationModel.setData(data);
+        qrCodeGenerationModel.setErrorCorrectionLevel("L");
+        qrCodeGenerationModel.setColorModel("black");
+        qrCodeGenerationModel.setTextFontName("Verdana");
+        qrCodeGenerationModel.setImageMargin(2);
```

Comment:
Instead of doing this in one line at a time, put all the defaults directly in the class. We will have less things to set then.
`private String fontName = "Verdana";`